pkgs/development/python-modules/augmax/default.nix

```diff
@@ -3,22 +3,29 @@
 , fetchFromGitHub
 , jax
 , jaxlib
+, pythonOlder
+, setuptools
```

Comment:
nit: i prefer to keep arguments a-z sorted in derivations i maintain

Reply:
Yes, of course I know that, but I accidentally forgot to sort by alphabetical order.
Thanks.